fastly/block_fastly_service_ratelimiter.go

```diff
@@ -267,9 +292,9 @@ func (h *RateLimiterAttributeHandler) Delete(_ context.Context, d *schema.Resour
 	return nil
 }
 
-func (h *RateLimiterAttributeHandler) createDeleteERLInput(service string, latestVersion int, resource map[string]any) gofastly.DeleteERLInput {
+func (h *RateLimiterAttributeHandler) createDeleteERLInput(rateLimiterID, service string, latestVersion int, resource map[string]any) gofastly.DeleteERLInput {
```

Comment:
I would just suggest editing this `createDeleteERLInput` method so it only accepts `rateLimiterID` as it doesn't appear to require any other args.